start_celery_worker.sh

```diff
@@ -0,0 +1 @@
+watchmedo auto-restart --directory=./ --pattern='*.py' --recursive -- celery --app=config.celery_app worker  --concurrency=1 --loglevel=INFO
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5765271)